pymongo/server.py

```diff
@@ -191,9 +192,9 @@ def run_operation(
             # Must publish in find / getMore / explain command response
             # format.
             if use_cmd:
-                res = docs[0]
+                res: _DocumentOut = docs[0]  # type: ignore[assignment]
             elif operation.name == "explain":
-                res = docs[0] if docs else {}
+                res = docs[0] if docs else {}  # type: ignore[assignment]
```

Comment:
Should this also be `res: _DocumentOut`?

Reply:
I believe that's implied here because of line 195

Reply:
Got it, thanks!